packages/xrpl/HISTORY.md

```diff
@@ -3,6 +3,9 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased
+### Added
+* Made unexpected errors in `submitAndWait` more verbose to make them easier to debug.
```

Comment:
I would call this a fix not an "added" - I think of the "added" section as new features (aka things that require a minor version bump, not a patch)